packages/app/src/app/overmind/namespaces/live/actions.ts

```diff
@@ -23,9 +23,10 @@ export const roomJoined: AsyncAction<{
   state.live.isLoading = false;
 });
 
-export const createLiveClicked: AsyncAction<{
-  sandboxId: string;
-}> = async ({ state, effects, actions }, { sandboxId }) => {
+export const createLiveClicked: AsyncAction<string> = async (
```

Comment:
Interesting, wouldn't this be less explicit (eg. I now need to know what's passed by going to the function call) and scalable (what if we want to add another option)? What's your opinion on this @christianalfoni?

Reply:
I did this in recommendation of @christianalfoni in an earlier PR, but can't find it anywhere tho 😕

Reply:
Yeah, I remember this was christian idea in new code as it makes it cleaner

Reply:
Yeah @christianalfoni said we should defo do it in new code.
But if I wanted, I could refactor them on the go

Can't find the comment where he said that anymore tho.
Too many PRs 😂😇

Reply:
Ah, yeah, there is no reason to pass an object anymore to actions. It was a requirement in Cerebral. 

You could argue that the typing does not explicitly say `sandboxId` anymore, but `payload`. I just favour reduced syntax as if you were to write this action from scratch you would type it with `string`, because of our lazyness as developers 😂